qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/test/test_gsas_handler.py

```diff
@@ -0,0 +1,322 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
I always forget to change the date here as well - it should be 2025 not 2024!

Reply:
Good point! I'll update these now. I was unsure if we still updated these or not. In Mantid Imaging we have stopped updating them like other open source libraries such as numpy.